README.md

```diff
@@ -1,3 +1,163 @@
 # MinioRunner
 
-Manages the installed minio binary and handles setup and teardown of the minio server.
+Manages the installed [minio](https://min.io/) binary and handles setup and
+teardown of the minio server.
+
+## Description
+
+This is a simple way of managing the locally installed minio server using an
+installed binary. This can be used to start/stop the server when running
+automated tests or developing locally against an S3 alternative. It also
+installs the [`mc` (minio client)](https://min.io/docs/minio/linux/reference/minio-mc.html) CLI
+tool to make interacting with the server easier.
+
+This an extremely focused gem and will not focus on all possible different
+configurations and binaries of minio. Only Linux and macOS platforms are
+supported at this time.
+
+This gem was inspired by the [webdrivers](https://github.com/titusfortner/webdrivers)
+project.
+
+## Usage
+
+In your Gemfile:
+
+```ruby
+ruby gem 'minio_runner', require: false
```

Comment:
```suggestion
gem 'minio_runner', require: false
```
?

Reply:
Weird, I probably just doubled up the `ruby` from the backticks, thanks!